### PR TITLE
[author] [new] add cPlayer v2.3.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by Copay**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):cPlayer(https://github.com/Copay/cPlayer)
 * Official website (optional, not the repository):http://cplayer.js.org
 * NPM package url (optional):https://www.npmjs.com/package/cPlayer
 * GitHub / Bitbucket popularity (required):
   - Count of stars:116
   - Count of watchers:7
   - Count of forks:16

# Essential checklist
 * [* ] I'm the author of this library
   * [* ] I would like to add link to the page of this library on CDNJS on website / readme
 * [* ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ * ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [* ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ * ] Has valid tags for each versions (for git auto-update)
 * [ * ] Auto-update setup
 * [ * ] Auto-update target/source is valid.
 * [ * ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

@PeterDaveHello Here it is,Thanks!

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@Copay Thanks. 
@pvnr0082t @naoh8103 @x09336 would you guys please take a look?

File: ajax/libs/cplayer/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  ],
+  "contributors": [
+    {
+      "name": "Corps",
+      "email": "zsq01@live.com",
+      "web": "https://corps.set-fire.com"
+    }
+  ],
+  "bugs": "zsq01@live.com",
```
Comment (pvnr0082t) [inline ajax/libs/cplayer/package.json]: @Copay Please remove maintainers, contrubutors, and bugs field.

**Comment by pvnr0082t** [comment]
And please add `filename` and `author' field.

File: ajax/libs/cplayer/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "license": "MIT",
+  "repository": {
+    "type":"git",
+    "url":"git+https://github.com/Copay/cPlayer.git"
+  },
+  "autoupdate":{
+    "source":"git",
+    "target":"git://github.com/Copay/cPlayer.git",
+    "basePath":"dist/",
```
Comment (pvnr0082t) [inline ajax/libs/cplayer/package.json]: We can just use `"dist"` here.

**Comment by Copay** [comment]
Is it ok?

File: ajax/libs/cplayer/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cPlayer",
```
Comment (pvnr0082t) [inline ajax/libs/cplayer/package.json]: This field should be the same as folder name. In this case, we should use `"cplayer"`.

File: ajax/libs/cplayer/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "keywords": [
+    "cPlayer",
+    "html5",
+    "audio",
+    "music",
+    "player",
+    "es6",
+    "cplayer"
+  ],
+  "author":"Corps",
```
Comment (pvnr0082t) [inline ajax/libs/cplayer/package.json]: There should be a space after colon mark. `filename` field should be also.

File: ajax/libs/cplayer/package.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type":"git",
+    "url":"git+https://github.com/Copay/cPlayer.git"
+  },
+  "autoupdate":{
+    "source":"git",
+    "target":"git://github.com/Copay/cPlayer.git",
+    "basePath":"dist",
+    "files":["**/*"]
+  }
```
Comment (pvnr0082t) [inline ajax/libs/cplayer/package.json]: Let's follow the format in [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).
```json
"autoupdate": {
    "source": "git",
    "target": "git://github.com/Copay/cPlayer.git",
    "basePath": "dist",
    "files": [
      "**/*"
    ]
}
```

**Comment by Copay** [comment]
Now?

File: ajax/libs/cplayer/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  "keywords": [
+    "cPlayer",
+    "html5",
+    "audio",
+    "music",
+    "player",
+    "es6",
+    "cplayer"
+  ],
+
```
Comment (PeterDaveHello) [inline ajax/libs/cplayer/package.json]: Please remove this empty line.

**Pull request merged**